public/app/features/canvas/elements/ellipse.tsx

```diff
@@ -23,26 +22,26 @@ class EllipseDisplay extends PureComponent<CanvasElementProps<EllipseConfig, Ell
   }
 }
 
-const getStyles = stylesFactory((theme: GrafanaTheme2, data) => ({
-  container: css`
-    display: table;
-    position: absolute;
-    top: 50%;
-    transform: translateY(-50%);
-    width: 100%;
-    height: 100%;
-    background-color: ${data?.backgroundColor};
-    border: ${data?.width}px solid ${data?.borderColor};
-    border-radius: 50%;
-  `,
-  span: css`
-    display: table-cell;
-    vertical-align: ${data?.valign};
-    text-align: ${data?.align};
-    font-size: ${data?.size}px;
-    color: ${data?.color};
-  `,
-}));
+const getStyles = (theme: GrafanaTheme2, data: any) => ({
+  container: css({
+    display: 'table',
+    position: 'absolute',
+    top: '50%',
+    transform: 'translateY(-50%)',
+    width: '100%',
+    height: '100%',
+    backgroundColor: data?.backgroundColor,
+    border: `${data?.width}px solid ${data?.borderColor}`,
+    borderRadius: theme.shape.radius.circle,
```

Comment:
this is set to 100% but still works the same as 50% value 🤷‍♂️

Reply:
@Develer

Reply:
@nmarrs with 50% height element doesn't match the container border.
See the comparison below.
<img width="323" alt="Screenshot 2023-10-12 at 19 27 45" src="https://github.com/grafana/grafana/assets/1418924/74ef4bdc-1933-4df6-9d88-3ae7f297573a">
<img width="332" alt="Screenshot 2023-10-12 at 19 27 25" src="https://github.com/grafana/grafana/assets/1418924/3f33c6be-0689-473a-b661-cf73b02f64be">